### PR TITLE
Update install-anp.yaml

### PR DESCRIPTION
**Pull request by aojea**
don't use the default queue number to avoid conflict with kindnet

### CODE REVIEW

File: install-anp.yaml [hunk 1]
```diff
@@ -80,7 +80,7 @@ spec:
       - name: kube-network-policies
         image: registry.k8s.io/networking/kube-network-policies:v0.6.0
         command: ["/bin/netpol"]
-        args: ["-v=2", "-admin-network-policy=true", "-baseline-admin-network-policy=true","--hostname-override=$(MY_NODE_NAME)"]
+        args: ["-v=2", "-admin-network-policy=true", "-baseline-admin-network-policy=true","--hostname-override=$(MY_NODE_NAME)","--nfqueue-id=99"]
```
Comment (tssurya) [inline install-anp.yaml:83]: lgtm, since default I guess is 0 what kube-net-pol uses will conflict with that kindnet uses so safer to use a custom value..

Comment (aojea) [reply install-anp.yaml:83]: default is 100

https://github.com/kubernetes-sigs/kube-network-policies/blob/d3eeb5525d5a47b8e4601ecc3476b120b0156992/cmd/main.go#L44

Comment (tssurya) [reply install-anp.yaml:83]: so kube-net-pol in KIND uses 100 and hence the conflict... when we install it separately.. I see

**Pull request merged**